Module3HM2_DelegatesEventsLinq/Extensions/ListExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Extensions
+{
+    public static class ListExtensions
+    {
+        public static T FirstOrDefault2<T>(this List<T> list, Func<T, bool> predicate)
```

Comment:
this IEnumerable, is usual target for extension, not IList, and not List (class)